tests/test_tasks.py

```diff
@@ -383,3 +403,85 @@ def ignore_call_to_send_delete_notice(self):
         (delete.send_delete_notice(
                 mox.IsA(tasks.CleanUpInTestMode), mox.IsA(model.Person)).
             MultipleTimes())
+
+
+class NotifyBadReviewStatusTests(unittest.TestCase):
+    handler = None
+
+    def setUp(self):
+        self.testbed = testbed.Testbed()
+        self.testbed.activate()
+        self.testbed.init_user_stub()
+        self.testbed.init_datastore_v3_stub()
+        self.testbed.init_memcache_stub()
+        self.testbed.init_mail_stub()
+        # root_path must be set the the location of queue.yaml.
+        # Otherwise, only the 'default' queue will be available.
+        path_to_app = os.path.join(os.path.dirname(__file__), '../app')
+        self.testbed.init_taskqueue_stub(root_path=path_to_app)
+
+        model.Repo(key_name='haiti').put()
+
+    def tearDown(self):
+        self.testbed.deactivate()
+
+    def get_handler(self):
+        if not self.handler:
+            self.handler = test_handler.initialize_handler(
+                handler_class=tasks.NotifyBadReviewStatus,
+                action=tasks.NotifyBadReviewStatus.ACTION,
+                repo='haiti', environ=None, params=None)
+        return self.handler
+
+    def clear_db_config(self):
+        [db.delete(entry) for entry in db.GqlQuery('SELECT * FROM ConfigEntry')]
+
+    def test_notify(self):
+        handler = self.get_handler()
+        self.assertEqual(handler._notify(10), False)
+        admin_email = 'inna-testing@gmail.com'
+        thres_unreviewed_notes = 10
+        config.set_for_repo('haiti', admin_user_email=admin_email)
+        config.set_for_repo('haiti', thres_unreviewed_notes=thres_unreviewed_notes)
+        self.assertEqual(handler._notify(10), True)
+        self.clear_db_config()
```

Comment:
You're right. But testbed will take more computational resources to initialize. I think it's not better to use testbed so many times since this will increase test time.